docarray/document/mixins/plot.py

```diff
@@ -70,38 +70,68 @@ def _plot_recursion(self, tree=None):
                     d._plot_recursion(_match_tree)
         return tree
 
-    def display(self):
-        """Plot image data from :attr:`.tensor` or :attr:`.uri`."""
-        from IPython.display import Image, display
+    def display(self, from_: Optional[str] = None):
```

Comment:
why is it with underscore ?

Reply:
`from` is a restricted keyword in python

Reply:
Okay I'm not sure what is the best practice here, @JoanFM  any idea ?